.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: go
+
+os:
+  - linux
+
+dist: bionic
+
+go:
+  - 1.14.x
+
+env:
+  global:
+    - GO111MODULE=on
+  jobs:
+    - GOTESTFLAGS=""
+    - GOTESTFLAGS="-race"
+
+script:
+  - cd interpreter_book/src
+  - make monkey
+  - make test
```

Comment:
will could also run linting in CI, maybe just gofmt for now?

Reply:
sure, feel free to add on top/change/remove stuff here :)

Reply:
Lets do it in a follow up PR once this is merged :heart: